imap_processing/hi/l1a/histogram.py

```diff
@@ -107,34 +108,35 @@ def allocate_histogram_dataset(num_packets: int) -> xr.Dataset:
         np.arange(2, 360, 4),
         name="angle",
         dims=["angle"],
-        attrs=hi_cdf_attrs.hi_hist_l1a_angle_attrs.output(),
+        attrs=cdf_manager.get_variable_attributes("hi_hist_angle"),
     )
     data_vars = dict()
     data_vars["ccsds_met"] = xr.DataArray(
         np.empty(num_packets, dtype=np.uint32),
         dims=["epoch"],
-        attrs=hi_cdf_attrs.ccsds_met_attrs.output(),
+        attrs=cdf_manager.get_variable_attributes("hi_hist_ccsds_met"),
     )
-    data_vars["esa_step"] = xr.DataArray(
+    data_vars["esa_stepping_num"] = xr.DataArray(
         np.empty(num_packets, dtype=np.uint8),
         dims=["epoch"],
-        attrs=hi_cdf_attrs.esa_step_attrs.output(),
+        attrs=cdf_manager.get_variable_attributes("hi_hist_esa_stepping_num"),
     )
 
-    for counter in (*QUALIFIED_COUNTERS, *LONG_COUNTERS, *TOTAL_COUNTERS):
-        data_vars[counter] = xr.DataArray(
+    default_counter_attrs = cdf_manager.get_variable_attributes("hi_hist_counters")
```

Comment:
Leave a docstring for what are qualified counters, long counters, etc. unless it is somewhere else in the code. I like how you used only one attribute  instead of duplicating these.